src/cirq_iqm/iqm_remote.py

```diff
@@ -0,0 +1,129 @@
+# Copyright 2020–2021 Cirq on IQM developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Implements a CIRQ compatible sampler that calls the IQM backend
+"""
+import os
+import cirq
+import numpy as np
+from cirq import study
+from cirq.study import resolver
+from cirq_iqm.iqm_client import IQMBackendClient, IQMCircuit, IQMInstruction
+
+
+def get_sampler_from_env() -> 'IQMSampler':
+    """
+    Initialize an IQM sampler using environment variables IQM_TOKEN and IQM_URL
+
+    Returns:
+        IQM Sampler
+    """
+    iqm_url = os.environ.get('IQM_URL')
```

Comment:
Is IQM_URL too unspecific? IQM_COCOS_URL? IQM_SERVER_URL? Something else? :)

Reply:
renamed